openid-zkp-auth/circuits/jwt_proof_ua.circom

```diff
@@ -0,0 +1,170 @@
+pragma circom 2.1.3;
+
+include "helpers/sha256.circom";
+include "helpers/misc.circom";
+include "helpers/strings.circom";
+include "helpers/zkhasher.circom";
+
+/**
+JWT Proof: User-agnostic (UA) circuit
+
+    Construction params:
+    - maxContentLength:         Maximum length of the JWT + SHA2 padding in bytes
```

Comment:
Please specify for both the implications on the size of the circuit

Reply:
Added